src/bucket/Bucket.cpp

```diff
@@ -89,6 +89,7 @@ Bucket::apply(Application& app) const
     ZoneScoped;
 
     BucketApplicator applicator(app, app.getConfig().LEDGER_PROTOCOL_VERSION,
+                                app.getConfig().LEDGER_PROTOCOL_VERSION,
```

Comment:
I'm questioning whether this is right. If we're applying an arbitrary bucket to the database, we need to actually load everything to ensure we don't crash right? If so, I think this should be `0`. A named constant with a comment would probably be best.

Reply:
Yeah good point. Fixed.